doc/enterprise/deployment.md

```diff
@@ -0,0 +1,117 @@
+
+Assuming followed one of our [deploy guides](http://pachyderm.readthedocs.io/en/latest/deployment/deploy_intro.html) and you have a Pachyderm cluster running, you should see that the state of your Pachyderm cluster is similar to the following:
+
+```
+$ kubectl get all
+NAME                        READY     STATUS    RESTARTS   AGE
+po/etcd-4197107720-br61m    1/1       Running   0          8m
+po/pachd-3548222380-s086m   1/1       Running   2          8m
+
+NAME             CLUSTER-IP     EXTERNAL-IP   PORT(S)                       AGE
+svc/etcd         10.111.11.36   <nodes>       2379:32379/TCP                8m
+svc/kubernetes   10.96.0.1      <none>        443/TCP                       10m
+svc/pachd        10.97.116.5    <nodes>       650:30650/TCP,651:30651/TCP   8m
+
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/etcd    1         1         1            1           8m
+deploy/pachd   1         1         1            1           8m
+
+NAME                  DESIRED   CURRENT   READY     AGE
+rs/etcd-4197107720    1         1         1         8m
+rs/pachd-3548222380   1         1         1         8m
+```
+
+Your `pachctl` CLI tool should also be able to connect to Pachyderm:
+
+```
+$ pachctl version
+COMPONENT           VERSION             
+pachctl             1.5.0               
+pachd               1.5.0
+```
+
+Once you have Pachyderm in this state, deploy Pachyderm Enterprise Edition is as simple as:
+
+```
+$ pachctl deploy local --dashboard-only
+``` 
+
+After a few minutes, you should see the new "dash" container running in Kubernetes:
+
+```
+$ kubectl get all
+NAME                        READY     STATUS    RESTARTS   AGE
+po/dash-3809689541-56tfb    2/2       Running   0          8m
+po/etcd-4197107720-jd9w5    1/1       Running   0          13m
+po/pachd-4280389576-qfcg8   1/1       Running   3          13m
+
+NAME             CLUSTER-IP   EXTERNAL-IP   PORT(S)                         AGE
+svc/dash         10.0.0.4     <nodes>       8080:30080/TCP,8081:30081/TCP   8m
+svc/etcd         10.0.0.152   <nodes>       2379:32379/TCP                  13m
+svc/kubernetes   10.0.0.1     <none>        443/TCP                         17m
+svc/pachd        10.0.0.193   <nodes>       650:30650/TCP,651:30651/TCP     13m
+
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/dash    1         1         1            1           8m
+deploy/etcd    1         1         1            1           13m
+deploy/pachd   1         1         1            1           13m
+
+NAME                  DESIRED   CURRENT   READY     AGE
+rs/dash-3809689541    1         1         1         8m
+rs/etcd-4197107720    1         1         1         13m
+rs/pachd-4280389576   1         1         1         13m
+```
+
+If you previously had port forwarding enabled for your Pachyderm cluster, you will also have to restart this forwarding:
+
+```
+$ ps aux | grep "forward"
```

Comment:
We have an open issue to have pachctl port-forward kill other port-forwards and then reconnect. It might be easy to ping the eng team to have that added cuz it's a really easy change. This would be a very nice convenience change. https://github.com/pachyderm/pachyderm/issues/1980